homeassistant/components/vallox/coordinator.py

```diff
@@ -0,0 +1,11 @@
+"""Coordinator for Vallox ventilation units."""
+
+from __future__ import annotations
+
+from vallox_websocket_api import MetricData
+
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+
+class ValloxDataUpdateCoordinator(DataUpdateCoordinator[MetricData]):
+    """The DataUpdateCoordinator for Vallox."""
```

Comment:
Why don't we move the coordinator + super init to here as well?

Reply:
This makes it a slightly bigger refactor, but I think it's ok.